packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -476,6 +476,7 @@ registerFragment(`
     }
     myEditorAccess
     linkSharingKey
+    lastRevisionAt
```

Comment:
Oh man... I really don't want this to be on `PostsPage`; that's inherited by nearly every single other relevant post fragment.  Maybe make a new fragment for use by `PostsEditPage` which extends `PostsPage` and just has this single additional field?